### ISSUE TITLE
Content editors can delete blocks in the Blocks Report

### ISSUE THREAD

**Issue by ekellagher** [CuBoulder/express_mono#598]
This is more a question of permissions intent or this could be an issue with masquerading as a user.

The Content Editor permissions say "Edit only access. A basic role limited to content edits/updates for existing content. Can’t delete or add content, blocks or layouts."
I was exploring the masquerading feature on https://www-test.colorado.edu/masquerade/
I chose Michael Weil who is a content editor
As Michael I could see the Reports tab and the unused blocks list
I switched back to myself and created an unused text block on the site
Then switched to Michael again and went to the Reports tab >Unused blocks
From here I could delete the the unused text block.

I did not realize before that this was possible and FYI we have had issues with clients accidentally deleting unused block content, then later realizing that the block was used off and on and it just happened to appear in the list when they were not actively using the content.

**Comment by kevincrafts** [comment]
Two things from this:

1. Anyone with permissions to view the list of blocks (admin/content/blocks) as access to this report. However, you cannot delete blocks from this report.  I tried this on the dev site and was not able to duplicate this behavior.

2. On the report we list the methods we used to create the list and say this block was not included in any entity references, or enabled contexts and advise to delete at your own risk.
